src/renderer/src/routes/pages/Send/TokenBalanceList.tsx

```diff
@@ -0,0 +1,78 @@
+import { Fragment } from 'react'
+import { useTranslation } from 'react-i18next'
+import { BlockchainIcon } from '@renderer/components/BlockchainIcon'
+import { BalanceHelper } from '@renderer/helpers/BalanceHelper'
+import { StringHelper } from '@renderer/helpers/StringHelper'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
+
+export const TokenBalanceList = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'send' })
+  const { accounts } = useAccountsSelector()
+  const balanceExchange = useBalancesAndExchange(accounts)
```

Comment:
Take this to the page and pass it here as a prop, because if you need it in another component, we don't need to call the accounts and balances hook again